micall/monitor/kive_watcher.py

```diff
@@ -263,7 +263,7 @@ def calculate_retry_wait(min_wait, max_wait, attempt_count):
 class KiveWatcher:
     def __init__(self,
                  config=None,
-                 result_handler=lambda result_folder: None,
+                 qai_upload_queue=Queue(),
```

Comment:
A [mutable value] for a default parameter is almost always a bad idea.

[mutable value]: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Good catch